<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>finance</class>
 <widget class="QMainWindow" name="finance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1751</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>finance</string>
  </property>
  <property name="styleSheet">
   <string notr="true">alternate-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="verticalWidget" native="true">
        <property name="styleSheet">
         <string notr="true">
QPushButton
{
   border:0px
}
QWidget{
background-color: #181745;
}
QPushButton:hover
{
   color:rgb(255, 255, 255);
}
QPushButton{
font: 25 10pt &quot;Nunito&quot;;
color: rgb(153, 153, 153)
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Financiére</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Accueil</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QGroupBox" name="groupBox_finance">
                <property name="title">
                 <string>Ajouter une transaction</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Mode de paiement</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="mode_line"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Type</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="type_line"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Categorie</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="cate_line"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Date de transaction</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateEdit" name="date_line">
                   <property name="dateTime">
                    <datetime>
                     <hour>0</hour>
                     <minute>0</minute>
                     <second>0</second>
                     <year>2024</year>
                     <month>1</month>
                     <day>1</day>
                    </datetime>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Montant</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="montant_line">
                   <property name="maximum">
                    <double>999999999.990000009536743</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QPushButton" name="update_finance">
                     <property name="text">
                      <string>Modifier</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="ajouter_line">
                     <property name="text">
                      <string>Ajouter</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="annuler">
                     <property name="text">
                      <string>Annuler</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="minimumSize">
                 <size>
                  <width>1000</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="title">
                 <string>Affichage</string>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Entrer le montant à rechercher </string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2" colspan="2">
                  <widget class="QPushButton" name="sortdate">
                   <property name="text">
                    <string>Trier par date</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0" colspan="2">
                  <widget class="QDoubleSpinBox" name="finance_montant">
                   <property name="maximum">
                    <double>99999999.989999994635582</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2" colspan="2">
                  <widget class="QPushButton" name="sortmontant">
                   <property name="text">
                    <string>Trier par montant</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0" colspan="4">
                  <widget class="QTableView" name="finance_table"/>
                 </item>
                 <item row="3" column="0">
                  <widget class="QPushButton" name="csv">
                   <property name="text">
                    <string>Exporter format Excel</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1" colspan="2">
                  <widget class="QPushButton" name="modifier_finance">
                   <property name="text">
                    <string>Modifier</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="3">
                  <widget class="QPushButton" name="supprimer">
                   <property name="text">
                    <string>Supprimer</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Bilan Financier</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Résultat financiére</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Total nombre de transaction</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="total_nb_finance">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>5</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Total</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="total_finance">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>100</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>ChatBot</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1751</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
